packages/govuk-frontend/src/govuk/settings/colours.unit.test.js

```diff
@@ -2,6 +2,35 @@ const { compileSassString } = require('@govuk-frontend/helpers/tests')
 
 describe('Organisation colours', () => {
   it('should define websafe colours that meet contrast requirements', async () => {
```

Comment:
**nitpick** As this test is for the new colours, should it be talking of 'contrast-safe' rather than 'websafe' colours?